webapp-django/crashstats/crashstats/templates/crashstats/report_list_base.html

```diff
@@ -9,6 +9,12 @@
 
     <div class="page-heading">
         <h2>Crash Reports for {{ signature }}</h2>
+        &nbsp;&nbsp;
+        <em>
+            <a href="{{ url('signature:signature_report') }}?{{ super_search_query_string }}">
+                See the equivalent Signature report page
+            </a>
+        </em>
```

Comment:
That's a lot of stuff that's best done with CSS. The `&nbsp;&nbsp;` for example should be `padding-left` and the `<em>` tag to wrap a `<a>` tag is best done with `font-style:italic`